go1.x-image/cookiecutter-aws-sam-hello-golang-lambda-image/{{cookiecutter.project_name}}/hello-world/Dockerfile

```diff
@@ -1,13 +1,7 @@
-FROM public.ecr.aws/bitnami/golang:1.16 as build-image
-
-WORKDIR /go/src
+FROM public.ecr.aws/docker/library/golang:1.19 as build-image
```

Comment:
This uses Go 1.19 but in the `go.mod` file it defines `go 1.18`. Should those match?